src/utils/ProcessManager.js

```diff
@@ -32,7 +32,7 @@ module.exports = class ProcessManager {
     this.messageContent = ''
     this.options = options
     this.limit = options.limit || 1900
-    this.splitted = content.match(new RegExp(`.{1,${this.limit}}`, 'gms')) || [' ']
+    this.splitted = content ? content.match(new RegExp(`.{1,${this.limit}}`, 'gms')) : [' ']
```

Comment:
```suggestion
    this.splitted = content.match(new RegExp(`.{1,${this.limit}}`, 'gms')) || [' ']
```

https://github.com/wonderlandpark/dokdo/blob/3b61669cff3713cb496e75a7016a38218f6092e0/src/utils/ProcessManager.js#L28

Will not cause error